akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerStressSpec.scala

```diff
@@ -43,11 +44,14 @@ object CircuitBreakerStressSpec {
       case Failure(ex: CircuitBreakerOpenException) ⇒
         circCount += 1
         breaker.withCircuitBreaker(job).pipeTo(self)
+      case Failure(_: TimeoutException) ⇒
+        timeoutCount += 1
+        breaker.withCircuitBreaker(job).pipeTo(self)
       case Failure(_) ⇒
         failCount += 1
         breaker.withCircuitBreaker(job).pipeTo(self)
       case GetResult ⇒
-        sender() ! Result(doneCount, failCount, circCount)
+        sender() ! Result(doneCount, timeoutCount, failCount, circCount)
```

Comment:
it is only checking the failCount here: https://github.com/akka/akka/pull/17596/files#diff-a6a84cc4febbb297157a42a8d412a9d0R78

but for debugging (println) it was good to see the distribution different things